File: Spring Boot/02_EmployeeCRUD/src/main/java/com/mindgate/main/domain/Employee.java
package com.mindgate.main.domain;

import java.time.LocalDate;

/**
 * @author MGS5237
 *
 */

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class Employee {
	private String employeeId;
	private String name;
	private double salary;
	private LocalDate dateOfBirth;
	private String departmentId;
	private String gender;
	private String state;
	private String emailId;

	

}

File: Angular/03-employee-crud/src/app/employee/domain/employee.ts
import { Department } from "./department";

export class Employee {
    employeeId: string = '';
    name: string = '';
    salary: number = 0.0;
    dateOfBirth: Date = new Date();
    departmentDTO: Department = new Department();
    gender: string = '';
    state: string = '';
    emailId: string = '';
}